deployment/terraform/batch-container-service.tf

```diff
@@ -0,0 +1,95 @@
+#
+# Security group resources
+#
+resource "aws_security_group" "batch_container_instance" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgBatchContainerInstance"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+#
+# Autoscaling Resources
+#
+data "template_file" "batch_container_instance_cloud_config" {
+  template = "${file("cloud-config/ecs-batch-user-data.yml")}"
+
+  vars {
+    ecs_cluster_name = "${aws_ecs_cluster.batch_container_instance.name}"
+    environment      = "${var.environment}"
+  }
+}
+
+resource "aws_launch_configuration" "batch_container_instance" {
+  lifecycle {
+    create_before_destroy = true
+  }
+
+  ebs_optimized = true
```

Comment:
I think that this will cause a failure if we ever try to use an instance type that doesn't support EBS optimizations (`t2.*`, etc.).